### DISCUSSION TITLE
Please add support for SwiftUI `View.modifier `

### DISCUSSION THREAD

**Discussion by shial4** [skiptools/skip#32] [General]
Please add support for SwiftUI Views modifier to pass custom modifier class in
Along with custom modifiers build of `ViewModifier`

I would love to SKIP enable this framework 
https://github.com/shial4/SwiftUIComponents/blob/main/Sources/SwiftUIComponents/Components/ViewModifiers/FrameModifier.swift

```
.modifier(FrameModifier(
         offset: CGSize(width: 10, height: 20),
         rotation: Angle(degrees: 45),
         scale: CGSize(width: 2, height: 2),
         anchor: .center)
     )
```

**Comment by marcprux** [comment]
Good suggestion, thanks. I've moved this over to https://github.com/skiptools/skip-ui/issues/3
